components/molecule/photoUploader/src/index.js

```diff
@@ -384,7 +392,7 @@ MoleculePhotoUploader.propTypes = {
 
   /**
    *  Callback that returns an array of files.
-   *  It's executed everytime an image is added, or is deleted, or is rotated, or is sorted
+   *  It's executed everytime an image is added, or is deleted, or is rotated, or is sorted, or it's label change
```

Comment:
```suggestion
   *  It's executed everytime an image is added, deleted, rotated, sorted or its label is changed
```

perhaps a bit more clear?